solr/solrj/src/java/org/apache/solr/common/ConfigNode.java

```diff
@@ -93,9 +107,16 @@ default String attr(String name) {
     return attributes().get(name);
   }
 
-  default String requiredStrAttr(String name, Supplier<RuntimeException> err) {
+  /**
+   * Like {@link #attr(String)} but throws an error (incorporating {@code missing_err}) if not
+   * found.
+   */
+  default String attrRequired(String name, String missing_err) {
```

Comment:
nit pick, but should this bee `missingErr`?   Thanks for the docs..

Reply:
Not my naming choice; I can rename

Reply:
Thanks!!!!